src/Vlingo.Actors.Tests/ProxyGeneratorTests.cs

```diff
@@ -17,47 +17,69 @@ public void ShouldIncludeNamespacesForMethodParameters()
         {
             var generator = ProxyGenerator.ForTest(false, World.DefaultLogger);
             var result = generator.GenerateFor(typeof(IProxyGenTestInterface));
+            Assert.Contains("private const string DoSomethingRepresentation1 = \"DoSomething(System.Threading.Tasks.Task)\";", result.Source);
             Assert.Contains("System.Threading.Tasks.Task t", result.Source);
+            Assert.Contains("Action<Vlingo.Actors.Tests.IProxyGenTestInterface> cons128873 = __ => __.DoSomething(t);", result.Source);
         }
 
         [Fact]
         public void ShouldNotIncludeGenericContraintForMethod()
         {
             var generator = ProxyGenerator.ForTest(false, World.DefaultLogger);
             var result = generator.GenerateFor(typeof(IProxyGenericMethodWithoutConstraint));
+            Assert.Contains("private const string WriteRepresentation1 = \"Write<TSource>(string, int)\";", result.Source);
             Assert.Contains("public void Write<TSource>(string id, int state)", result.Source);
+            Assert.Contains("Action<Vlingo.Actors.Tests.IProxyGenericMethodWithoutConstraint> cons128873 = __ => __.Write<TSource>(id, state);", result.Source);
```

Comment:
Aren't those numbers randomly generated ? Just asking because I'm not sure.

Reply:
`cons128873` is fixed in the code.